_includes/forms/amending-and-correcting-records-under-the-privacy-act-form.html

```diff
@@ -0,0 +1,168 @@
+            </label>
+            <input class="input__long"
+                   id="organization-city"
+                   type="text"
+                   required>
+        </div>
+
+        <div class="form-l_col form-l_col-1-2">
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header"
+                       for="organization-state">
+                    State
+                </label>
+                <div class="custom-select">
+                    <select class="custom-select_select"
+                            data-placeholder="Select state"
+                            id="organization-state"
+                            required>
+                        <option value="">Select state</option>
+                        <option value="AL">Alabama</option>
+                        <option value="AK">Alaska</option>
+                        <option value="AZ">Arizona</option>
+                        <option value="Other">Other States to be added</option>
+                    </select>
+                    <span class="custom-select_icon
+                                 cf-icon
+                                 cf-icon-down"></span>
+                </div>
+            </div>
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="organization-zip">
+                    Zip code
+                </label>
+                <input class="input__long"
+                       id="organization-zip"
+                       type="text"
+                       required>
+            </div>
+        </div>
+        <div class="form-l_col form-l_col-1-2">
+            <label class="form-label-header" for="telephone-number">
+                Telephone number
+            </label>
+            <input class="input__long"
+                   id="telephone_number"
+                   type="number"
+                   required>
+        </div>
+        <div class="form-l_col form-l_col-1-2">
+            <label class="form-label-header" for="contact-time">
+                Best time to reach you
+            </label>
+            <div class="custom-select">
+                <select class="custom-select_select"
+                        data-placeholder="Select Time"
+                        id="contact-time">
+                      <option value="">Select time</option>
+                      <option>8:00 a.m</option>
+                 </select>
+                 <span class="custom-select_icon
+                                 cf-icon
+                                 cf-icon-down"></span>
+            </div>
+        </div>
+        <div class="form-l_col form-l_col-1-2">
+            <label class="form-label-header"
+                   for="fax-number">
+                Fax number
+            </label>
+            <input class="input__long"
+                   id="fax-number"
+                   type="text"
+                   placeholder="(Optional)">
```

Comment:
Can we move this to part of the label? Screenreaders and older browsers won't be able to see the optional. [Ex](https://github.com/cfpb/cfgov-refresh/blob/flapjack/_includes/templates/forms/plain-writing-feedback-2-form.html#L109)